yt_dlp/extractor/urplay.py

```diff
@@ -14,12 +14,13 @@ class URPlayIE(InfoExtractor):
     _VALID_URL = r'https?://(?:www\.)?ur(?:play|skola)\.se/(?:program|Produkter)/(?P<id>[0-9]+)'
     _TESTS = [{
         'url': 'https://urplay.se/program/203704-ur-samtiden-livet-universum-och-rymdens-markliga-musik-om-vetenskap-kritiskt-tankande-och-motstand',
-        'md5': 'ff5b0c89928f8083c74bbd5099c9292d',
+        'md5': '5ba36643c77cc3d34ffeadad89937d1e',
```

Comment:
It is not normal for md5 to change. Make sure to actually check that this video downloads correctly

Reply:
I have no idea. Without any other modifications - i.e. from yt-dlp/master - these testcases consistently fail. Even if I VPN into the country of origin, i.e. Sweden.

Downloading the test cases' media works for me.